tests/integration/test_ontology.py

```diff
@@ -6,6 +6,14 @@
 import time
 
 
+def test_delete_tool_feature_from_ontology(client, ontology):
+    feature_schema_to_delete = ontology.normalized['tools'][0]
+    result = client.delete_feature_schema_from_ontology(
+        ontology.uid, feature_schema_to_delete['featureSchemaId'])
+    assert result['deleted'] == True
+    assert result['archived'] == False
```

Comment:
Using my ontology fixture to test deleting a node.

Reply:
Great! It would be good to assert that the length of tools is 2 prior to deletion, just for readability.